test/integration/golden_response/get_statvar_group/uk.json

```diff
@@ -6706,7 +6708,7 @@
           "specializedEntity": "Stock"
         }
       ],
-      "numDescendentStatVars": 2628
+      "numDescendentStatVars": 217523
```

Comment:
This number is very suspicious

Reply:
This is not using the existence cache yet. Going to send a PR for it.